steam_wrapper/steam_wrapper.py

```diff
@@ -351,6 +353,12 @@ def configure_shared_library_guard():
     else:
         os.environ["LD_AUDIT"] = f"/app/links/$LIB/libshared-library-guard.so"
 
+def clear_empty_environment(current_info):
+    environment = current_info["environment"]
+    for key, value in os.environ.items():
+        if environment.get(key) == "":
+            logging.debug(f"Ensuring {key} not in environment")
```

Comment:
I'd rather word it "Removing environment variable {key}". I mean, just say what we do, not why we do it.

Reply:
I'm going to see if Flathub supports --unset-env. It allows doing this thing with no wrapper changes.